tests/jballerina-integration-test/src/test/resources/configurables/negative_tests/additionalField/Config.toml

```diff
@@ -0,0 +1,4 @@
+[main.testUser]
```

Comment:
Shall we not have folder names in camel case?

Reply:
Shall we fix in all places?

Reply:
I thought project names can be in camel case. There were some other tests with project name like `testProject`. 
Do we need to use `_` instead?

Reply:
Normally camel case is not preferred as folder names. If it is okay in ballerina then that is fine